files/default/queued_job_count.py

```diff
@@ -18,19 +18,22 @@ def main(args):
     # then get their running operations
     running_ops = []
     for wf in running_wfs:
-        running_ops.extend(filter(
-            lambda x: x.state in ["RUNNING","WAITING"],
-            wf.operations
-        ))
+        running_ops.extend([
+            x for x in wf.operations
+            if x.state in ["RUNNING", "WAITING"]
```

Comment:
Where does WAITING come from? Is this related to Greg's future job weights code?